identity-service/src/notifications/formatNotificationMetadata.js

```diff
@@ -227,15 +229,15 @@ const RemixCosignTitle = 'New Track Co-Sign! 🔥'
 const challengeInfoMap = {
```

Comment:
Is there a way to have these amount value come from the other config? probably better to not multiple places to change

Reply:
Was talked offline, it's set as another linear task, but requires more changes